WordPress/Classes/Models/PostCategory+Creation.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension PostCategory {
+
+    static func create(withBlogID id: NSManagedObjectID, in context: NSManagedObjectContext) throws -> PostCategory {
+        let object = try context.existingObject(with: id)
+
+        guard let blog = object as? Blog else {
+            fatalError("The object id does not belong to a Blog: \(id)")
```

Comment:
Did you consider inlining this?

```suggestion
        guard let blog = try context.existingObject(with: id) as? Blog else {
            fatalError("The object id does not belong to a Blog: \(id)")
```

Reply:
I wanted to be _super_ clear what the "fatal error" here is, it's the casting not `existingObject(with:)` the throwing function. I know errors from the throwing function won't be captured by the `guard`, but still, my dumb head requires me to double check to make sure what exactly the `else` case is here.

Reply:
Gotcha. Sounds good.